Source/Krypton Components/Krypton.Toolkit/General/CommonHelperThemeSelectors.cs

```diff
@@ -0,0 +1,120 @@
+﻿#region BSD License
+/*
+ *
+ * Original BSD 3-Clause License (https://github.com/ComponentFactory/Krypton/blob/master/LICENSE)
```

Comment:
Remove original BSD Compoenent license part from new file.

Start date for new BSD should be the year created (in this case 2024)

Reply:
> mpoenent license part from new file.
> 
> Start date for new BSD should be the year created (in this c

A lot of source files have this Original clause....

Reply:
Original Files - Of course - That is the rule(s) for BSD
New files - have the new start dates etc.

Reply:
Corrections posted